testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/jsonb/basic/JsonBindingDebugLoggingTest.java

```diff
@@ -0,0 +1,325 @@
+    * up the call stack.  This exception is passed through undertow code which adds
+    * HTML formatting to the message that is returned to the caller.
+    */
+   @Test
+   public void exceptionDuringServerSend() throws Exception {
+      // count log messages before request
+      LogCounter errorStringLog = new LogCounter("ERROR",
+              false, ContainerConstants.DEFAULT_CONTAINER_QUALIFIER);
+
+      LogCounter resteasyExceptionLog = new LogCounter(
+              ".*ERROR .* RESTEASY002025.*", true,
+              ContainerConstants.DEFAULT_CONTAINER_QUALIFIER, true);
+
+      LogCounter yassonExceptionLog = new LogCounter(
+              "Caused by: javax.json.bind.JsonbException",
+              true, ContainerConstants.DEFAULT_CONTAINER_QUALIFIER);
+      LogCounter yassonStacktraceLog = new LogCounter(
+              "at org.eclipse.yasson", true,
+              ContainerConstants.DEFAULT_CONTAINER_QUALIFIER);
+
+      LogCounter applicationExcpetionLog = new LogCounter(
+              "Caused by: java.lang.RuntimeException: "
+            + JsonBindingDebugLoggingItemCorruptedGet.class.getSimpleName(),
+              true, ContainerConstants.DEFAULT_CONTAINER_QUALIFIER);
+
+      // perform request
+      WebTarget base = client.target(generateURL("/get/nok"));
+      Response response = base.request().get();
+
+      // check response
+      Assert.assertThat("Wrong response code", response.getStatus(), is(500));
+      Assert.assertThat("Response message doesn't contains full stacktrace",
+              response.readEntity(String.class), allOf(
+            containsString("org.eclipse.yasson.internal"),
+            containsString("java.lang.RuntimeException: "
+                              + JsonBindingDebugLoggingItemCorruptedGet.class.getSimpleName()),
+            containsString("javax.json.bind.JsonbException: Error getting value on"),
+            containsString("RESTEASY008205")
+      ));
+
+      // checking server.log contents
+      // For applicationExcpetionLog and yassonExceptionLog there are
+      // always 2 log records for this text.  One is generated from
+      // resteasy and one is generated from undertow.
+      // assert log messages after request
+      Assert.assertThat("Application Exception should be logged",
+              applicationExcpetionLog.count(), is(2));
+      Assert.assertThat("RESTEasy exception should be logged",
+              resteasyExceptionLog.count(), is(1));
+      Assert.assertThat("Yasson exception should be logged",
+              yassonExceptionLog.count(), is(2));
+      Assert.assertThat("Yasson exception stacktrace should be logged",
+              yassonStacktraceLog.count(), greaterThan(0));
+
+      // There is always 2 log records for this error.  One is generated
+      // from resteasy and one is generated from undertow.
+      Assert.assertThat("There are not only 2 error logs in server",
+              errorStringLog.count(), is(2));

```

Comment:
AFAICS, errors are still printed with ERROR log level
```
15:10:28,795 ERROR [org.jboss.resteasy.resteasy_jaxrs.i18n] (default task-1) RESTEASY002025: Unknown exception while executing GET /get/nok: javax.ws.rs.ProcessingException: RESTEASY008205: JSON Binding serialization error javax.json.bind.JsonbException: Error getting value on: org.resteasy.simple.deployment.ItemCorruptedGet@4b51834d
...
```

Wouldn't be better debug logs?

Reply:
It is also not aligned with server-receive use-case: https://github.com/resteasy/Resteasy/pull/1889/files#diff-de9b68d9aa28165e02d5bdc6d92955f2R201

Reply:
No "RESTEASY002025: Unknown exception" should NOT be DEBUG.  This is used for any unknown exception.  It should remain an is.